### DISCUSSION TITLE
Issues View

### DISCUSSION THREAD

**Discussion by warhorus** [gotson/komga#1153] [Ideas]
For readers who are working on large shared universes that span multiple titles (Marvel or DC, for example), it might be useful to be able to view a library by issues instead of by series. This would enable them to filter down to unread issues within the shared universe, then sort by publication date, allowing them to easily read the oldest unread issue.

As it stands, they could maintain a reading list for the same purpose, but doing so is definitely an extra layer of effort for the users that this would eliminate.

**Comment by gotson** [comment]
It's been suggested before but there's no plan for it yet.

**Reply by warhorus** [reply]
I'd be happy to try giving a poke at it, if that wouldn't interfere with your roadmap.

**Reply by gotson** [reply]
Will be in the next release

**Comment by warhorus** [comment]
In case anyone down the line is looking for this feature, I've got a fork over here that's got it cooked in:

https://github.com/warhorus/komga
https://hub.docker.com/r/warhorus/komga

Just look at the top right of your library view for a new icon that lets you toggle between "Book" and "Series". I'm going to keep it up to date with Komga as best I can otherwise.

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.20.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.20.0))
